cmd/acr/purge.go

```diff
@@ -484,11 +504,7 @@ func dryRunPurge(ctx context.Context, acrClient api.AcrCLIClientInterface, login
 			for _, tag := range *tagsToDelete {
 				// For every tag that would be deleted first check if it exists in the map, if it doesn't add a new key
 				// with value 1 and if it does just add 1 to the existent value.
-				if _, exists := deletedTags[*tag.Digest]; exists {
-					deletedTags[*tag.Digest]++
-				} else {
-					deletedTags[*tag.Digest] = 1
-				}
+				deletedTags[*tag.Digest] += 1
```

Comment:
Is the change related to the reg issue?

Reply:
Golang linter doesn't didn't like this so I just fixed it so it would stop complaining